lib/hammer_cli_foreman_virt_who_configure/config.rb

```diff
@@ -70,6 +70,9 @@ class InfoCommand < HammerCLIForeman::InfoCommand
           field :hypervisor_server, _('Hypervisor server')
           field :hypervisor_username, _('Hypervisor username')
           field :kubeconfig_path, _('Configuration file'), Fields::Field, :hide_blank => true
+          field :prism_flavor, _('AHV Prism flavor'), Fields::Field, :hide_blank => true
+          field :ahv_update_interval, _('AHV update frequency'), Fields::Field, :hide_blank => true
+          field :ahv_internal_debug, _('Enable AHV debug'), Fields::Field, :hide_blank => true
```

Comment:
I'd recommend to use `Fields::Boolean` for this as well as we already have here: https://github.com/theforeman/hammer-cli-foreman-virt-who-configure/blob/b75a2d4f9cc5237cbc962bbe07dc9bfc7dd67af2/lib/hammer_cli_foreman_virt_who_configure/config.rb#L90

Reply:
Nice catch!